Update query and setting API endpoints for SQL and PPL documentations

chloe-zh opened the pull request:
Signed-off-by: chloe-zh <chloezh1102@gmail.com>

### Description
- Updated the endpoints:
  - `_opensearch/_sql` -> `_plugins/_sql`, related PR: https://github.com/opensearch-project/sql/pull/66
  - `_opensearch/_ppl` -> `_plugins/_ppl`, related PR: https://github.com/opensearch-project/sql/pull/66
  - `_opensearch/_sql/settings` and `_opensearch/_ppl/settings` -> `_plugins/_query/settings`, related PR: https://github.com/opensearch-project/sql/pull/82

- Removed deprecated settings including:
  - `opendistro.sql.cursor.enabled`, related PR: https://github.com/opensearch-project/sql/pull/75
  - `opendistro.sql.cursor.fetch_size`, related PR: https://github.com/opensearch-project/sql/pull/75
  - `opensearch.sql.query.analysis.*`, related PR: https://github.com/opensearch-project/sql/pull/76
  - `opensearch.sql.query.response.format setting`, related PR: https://github.com/opensearch-project/sql/pull/77

- Added settings:
  - `plugins.sql.delete.enabled`, related PR: https://github.com/opensearch-project/sql/pull/81

- Other minor changes:
  - Corrected the default format from json to JDBC
  - Corrected some table results that present improperly
  - Corrected PPL `head` command syntax
 
### Issues Resolved
#23 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

aetter commented on docs/ppl/commands.md:
```diff
@@ -617,7 +590,7 @@ search source=accounts | rare age by gender;
 | M  | 32
 | M  | 33
 
-## top {#top-command}
```
Hi @chloe-zh, this actually replaces the auto-generated ID of `#top` with `#top-command` so that the righthand navigation takes you to the proper section rather than the top of the webpage. It's worth keeping.

chloe-zh replied:
Ok I see, sounds good I'll add it back, thanks!

aetter commented on docs/sql/workbench.md:
```diff
@@ -1,14 +1,14 @@
+# Query Workbench
 
-Use the SQL workbench to easily run on-demand SQL queries, translate SQL into its REST equivalent, and view and save results as text, JSON, JDBC, or CSV.
+Use the Query workbench to easily run on-demand SQL queries, translate SQL into its REST equivalent, and view and save results as text, JSON, JDBC, or CSV.
```
Query Workbench

aetter reviewed (approved):
Looks awesome, thanks for all the work @chloe-zh. Two nitpicks. Should this be merged ASAP, or wait for RC?

chloe-zh commented:
> Looks awesome, thanks for all the work @chloe-zh. Two nitpicks. Should this be merged ASAP, or wait for RC?

@aetter Just addressed the comments. The plugin changes are along with the rc-1 changes, so I think it's fine to wait for RC.
